<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bouton_evenement</class>
 <widget class="QDialog" name="bouton_evenement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>departement evenement</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>661</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>Evenement</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
 #MyLoginForm {
background: red;
}

#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid red;
border-radius: 8px;
}

QPushButton {
color: white;
  background: bleu;

border-width: 6px;
border-color: grey;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 150px;
max-width: 150px;
min-height: 35px;
max-height: 35px;
}
QLabel {
     border: 4px solid black;
     border-radius: 12px;
     padding: 0 12px;
     background: red;
     selection-background-color: black;
 }
QPushButton:hover { background-color: #FF0101; }
 
  
</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Créer</string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>190</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>sujet</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nomev">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_dateev">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_sujetev">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>190</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_inserev">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>100</y>
         <width>172</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string>Insérer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Consulter</string>
      </attribute>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>361</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>40</y>
         <width>172</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string>tri selon le nom</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>120</y>
         <width>172</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string>tri selon la date</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>200</y>
         <width>172</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string>tri selon le sujet</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>modifier</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_modifierev_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>220</y>
         <width>172</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>299</x>
         <y>25</y>
         <width>231</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="verticalSpacing">
         <number>40</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>nom</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>date</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>sujet</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_nomev_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_dateev_2"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_sujetev_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableView_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>281</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Eliminer</string>
      </attribute>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>90</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supprimerev">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>90</y>
         <width>172</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>rechercher</string>
      </attribute>
      <widget class="QTableView" name="tableView_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>11</y>
         <width>371</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nomev_3">
       <property name="geometry">
        <rect>
         <x>79</x>
         <y>60</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nomev_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Sujet</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nomev_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Réclamation</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
